### PR TITLE
DEVTOOLS-68: Display new/deleted folders in status command.

### PR DESCRIPTION
**Pull request by juancoen**

### CODE REVIEW

File: lib/services/localService.js [hunk 1]
```diff
@@ -27,37 +27,64 @@ module.exports = function (apiPlatformRepository, BPromise,
...
+              storedDirectories = _.reject(storedDirectories, 'path',
+              localDirectoryPath);
+            } else {
+              result.addedDirectories.push(localDirectoryPath);
+            }
+          });
+
+          result.deletedDirectories = _.pluck(storedDirectories, 'path');
+
+          return BPromise.resolve();
```
Comment (fschargorosdky) [inline lib/services/localService.js]: There is no need for a promise here.
You can use Promise.all with an array that mixes promises and values, even with all values

**Comment by fschargorosdky** [comment]
Update the tests for getStatus(). Also change the test so it asserts exactly which attributes the result must have.

File: lib/services/localService.js (hunk 1 above)
Comment (juancoen) [inline lib/services/localService.js]: done

**Comment by fschargorosdky** [comment]
In the getStatus test, change how the result is asserted to use a deepEqual comparison:
```js
should.deepEqual(result, expectedResult)
```
that way it's safer (can't add new properties without breaking the test) and it's shorter to write

**Comment by juancoen** [comment]
done

**Comment by fschargorosdky** [comment]
Looks good to me

**Pull request merged**